demisto_sdk/commands/common/hook_validations/image.py

```diff
@@ -61,21 +65,34 @@ def is_valid(self):
 
         return self._is_valid
 
-    def validate_size(self, allow_empty_image_file: bool, maximum_size: int = IMAGE_MAX_SIZE) -> None:
+    def validate_size(self,
+                      allow_empty_image_file: bool,
+                      maximum_size: int = IMAGE_MAX_SIZE,
+                      should_validate_dimensions: bool = False,
+                      allowed_width: int = IMAGE_WIDTH,
+                      allowed_height: int = IMAGE_HEIGHT) -> None:
         """
         Checks if image has a valid size.
         if 'allow_empty_image_file' is true, checks that the image file is not empty.
         Args:
             allow_empty_image_file (bool): Whether empty image file is an error.
             maximum_size (int): Maximum allowed size.
+            should_validate_dimensions (bool): Should validate the image dimensions (Not being validated on AuthorImages)
+            allowed_height (int): the allowed height of the image
+            allowed_width (int): the allowed weight of the image
         """
         if re.match(IMAGE_REGEX, self.file_path, re.IGNORECASE):
             image_size = os.path.getsize(self.file_path)
             if image_size > maximum_size:  # disable-secrets-detection
                 error_message, error_code = Errors.image_too_large()
                 if self.handle_error(error_message, error_code, file_path=self.file_path):
                     self._is_valid = False
-
+            if should_validate_dimensions:
+                width, height = imagesize.get(self.file_path)
```

Comment:
What happens if the file is not really an image, but just with an image suffix?

Reply:
@DeanArbel 
If there is no image or corrupted one the library will return (-1, -1), therefore it will fail the validation.
Anyway, there is already a check for `is_existing_image` in the same class.